content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Indexed Job for Parallel Processing with Static Work Assignment
+content_type: task
+min-kubernetes-server-version: v1.21
+weight: 30
+---
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- overview -->
+
+{{< note >}}
+To be able to create Indexed Jobs, make sure to enable the `IndexedJob`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](docs/reference/command-line-tools-reference/kube-apiserver/)
+and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
+{{< /note >}}
+
+In this example, we will run a Kubernetes Job with multiple parallel worker processes.
+Each worker Pod gets assigned a unit of work statically, which is determined by
+the pod index available in the {{< glossary_tooltip text="annotation" term_id="annotation" >}}
+`batch.alpha.kubernetes.io/job-completion-index`, also accessible to containers
+in the `JOB_COMPLETION_INDEX` environment variable.
+
+Here is an overview of the steps in this example:
+
+1. **Create an image that can read the pod index**
+2. **Start an Indexed Job**. The downward API allows you to pass the annotation
+   as an environment variable or file to the container.
+
```

Comment:
This needs a prerequisites (“Before you begin”) section. See https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#before-you-begin for an example (refer to `content/en/docs/tasks/inject-data-application/distribute-credentials-secure.md`).

For example:
```markdown
## {{% heading "prerequisites" %}}

{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}

This task uses the Docker command line tool to build a container image. 
You also need write access to a container image registry that can host the image
you build.

If you do not have Docker available on your computer, you should install it.
You can instead use a Docker-compatible tool to build container images from a 
`Dockerfile`, such as Podman. You would then need to change `docker` in the
examples to match the name of the tool you are using.
```

Reply:
Added, except the docker part for now.